pennylane/gradients/finite_difference.py

```diff
@@ -161,7 +161,7 @@ def generate_shifted_tapes(tape, idx, shifts, multipliers=None):
         ``idx`` shifted by consecutive values of ``shift``. The length
         of the returned list of tapes will match the length of ``shifts``.
     """
-    params = tape.get_parameters()
+    params = list(tape.get_parameters())

```

Comment:
How come this was changed?

Reply:
Oh, I discovered during this PR that sometimes `tape.get_parameters()` will return an Autograd `SequenceBox`, for some reason. I was unable to work out what causes this to happen, but making sure that params is always cast to a list fixes it (for now).

I might be able to better track it down once the new interfaces are finished